### PR TITLE
Add ceph integration to glance

### PR DESCRIPTION
**Pull request by hemanthnakkina**
Add microceph integration to glance.
Use count to enable/disable the integration.

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -53,3 +53,13 @@ variable "config" {
   description = "Set configuration on model"
   default     = {}
 }
+
+variable "enable_ceph" {
+  description = "Enable ceph intergrations"
```
Comment (javacruft) [inline variables.tf:58]: "Enable Ceph integration"

Comment (hemanthnakkina) [reply variables.tf]: Done

File: variables.tf [hunk 2]
```diff
@@ -53,3 +53,13 @@ variable "config" {
   description = "Set configuration on model"
   default     = {}
 }
+
+variable "enable_ceph" {
+  description = "Enable Ceph intergration"
```
Comment (javacruft) [inline variables.tf:58]: 'integration'

Comment (hemanthnakkina) [reply variables.tf:58]: Done

**Pull request merged**